inspirehep/modules/hal/tei.py

```diff
@@ -32,9 +32,10 @@
 from inspirehep.utils.record_getter import get_es_records
 
 from .templates import template
+from .typology import hal_doctype, doctype_fallback
 
 
-def tei_response(record):
+def tei_response(record, has_attachment=False):
```

Comment:
What about calling it `json2tei`? or `convert_record_to_tei`. `tei_response` seems like the name of a variable containing the response from a service called `tei` :-)